server/tools/jgroups.sh

```diff
@@ -27,4 +27,19 @@ if [ -n "$JGROUPS_DISCOVERY_PROTOCOL" ]; then
     else
        $JBOSS_HOME/bin/jboss-cli.sh --file="/opt/jboss/tools/cli/jgroups/discovery/default.cli" >& /dev/null
     fi
+
```

Comment:
This shouldn't be invoked from `jgroups.sh` file. Perhaps it should be in a separate file - `infinispan.sh`?

Reply:
Certainly an easy change. But does it make sense to configure replicas without jgroups? I thought the intention was to trigger the clustering that way.

But fair enough, I'll update the PR if we agree it makes sense. What to trigger this new .sh file then?

Reply:
Reading the documentation about clustering (https://github.com/keycloak/keycloak-containers/blob/master/server/README.md#clustering) again, the jgroup part is only about adjusting how clustering works, but the multicast discovery seems to be enabled by default ... thus the issue https://issues.jboss.org/browse/KEYCLOAK-11009 of not actually being able to "disable" it. So externalizing the configuration snippets for replicasd out of the jgroups configuration makes total sense now.

Reply:
Yes, this is exactly the point. Default discovery protocols might be good enough for some cases.